src/wallet/init.cpp

```diff
@@ -124,10 +126,6 @@ bool WalletInit::ParameterInteraction() const
     if (gArgs.GetArg("-prune", 0) && gArgs.GetBoolArg("-rescan", false))
         return InitError(_("Rescans are not possible in pruned mode. You will need to use -reindex which will download the whole blockchain again."));
 
-    if (::minRelayTxFee.GetFeePerK() > HIGH_TX_FEE_PER_KB)
-        InitWarning(AmountHighWarn("-minrelaytxfee") + " " +
```

Comment:
The warning was added as a wallet warning in #8486.

This PR is meant to be a pure refactor (except for the updated log message) so I'm not going to add the suggested new warning for a high `-minrelaytxfee` to the node, but agree that it could make sense to add it in a separate PR. I'd also suggest that in a follow-up PR we change this warning to only trigger if `-minrelayfee` is greater than `walletInstance->m_default_max_tx_fee`.

Reply:
EDIT: these warnings will now show for each wallet at startup rather than just once for the node, so that's also a minor behaviour change when starting with multiple wallets.